README.md

```diff
@@ -5,3 +5,4 @@ A web application to manage OKR.
 ## Resources
 
 1. :memo: Introduction to OKR - [Link](https://www.oreilly.com/business/free/files/introduction-to-okrs.pdf)
+[![CircleCI](https://circleci.com/gh/fs-101/okr-manager.svg?style=svg)](https://circleci.com/gh/fs-101/okr-manager)
```

Comment:
Move badge to the top. See this demo repo https://github.com/CircleCI-Public/circleci-demo-ios

Reply:
done